lib/project_types/rails/commands/serve.rb

```diff
@@ -31,7 +31,12 @@ def call(*)
           env = ShopifyCli::Project.current.env.to_h
           env.delete('HOST')
           env['PORT'] = ShopifyCli::Tunnel::PORT.to_s
-          @ctx.system('bin/rails server', env: env)
+          env['GEM_PATH'] = Gem.gem_path(@ctx)
+          if @ctx.windows?
+            @ctx.system("ruby bin\\rails server", env: env)
```

Comment:
does `bin\rails` need to be prefixed by ruby in windows? If not, could we just use `File.join('bin', 'rails')`?

Reply:
`bin/rails` on Windows isn't an executable as far as Windows is concerned. Windows expects a command to end with `.exe` or `.bat` or `.cmd`, etc., so that's why it needs to be prefixed with `ruby`